rotkehlchen/api/rest.py

```diff
@@ -2231,7 +2233,7 @@ def _get_ethereum_airdrops(self) -> Dict[str, Any]:
                 addresses=self.rotkehlchen.chain_manager.accounts.eth,
                 data_dir=self.rotkehlchen.data_dir,
             )
-        except RemoteError as e:
+        except (RemoteError, WriteError, InvalidData) as e:
```

Comment:
separate those. `RemoteError` and `UnableToDecryptRemoteData`  should return a BAD_GATEWAY. That's because they both depend on external data and the connection to get them.

`OSError` is a problem with the user's system. Now the question is what status code to return here. Probably a `501`.
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#5xx_server_errors

We need to think of local system errors and error codes but probably a 501 for now is good. You should also write an entry about the new status code in the rest api docs for the airdrops.